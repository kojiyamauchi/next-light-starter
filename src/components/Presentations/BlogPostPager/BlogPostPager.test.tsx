/*
  Blog Post Pager Component Unit Test.
*/

import { BlogPostPager } from './'
import { render /* ,screen */ } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
// import userEvent from '@testing-library/user-event'

describe('Blog Post Pager Component Unit Test', () => {
  it('Snap Shot Testing', () => {
    const { asFragment } = render(<BlogPostPager prev="2" next="3" />)
    expect(asFragment()).toMatchSnapshot()
  })
})
