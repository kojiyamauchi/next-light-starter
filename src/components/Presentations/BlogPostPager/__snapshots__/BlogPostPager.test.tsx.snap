// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Blog Post Pager Component Unit Test Snap Shot Testing 1`] = `
<DocumentFragment>
  <ul
    class="wrapper"
  >
    <li
      class="prev"
    >
      <a
        class="prev-link"
        href="/blog-post/2"
        rel="prev"
      >
        New Post.
      </a>
    </li>
    <li
      class="next"
    >
      <a
        class="next-link"
        href="/blog-post/3"
        rel="prev"
      >
        Old Post.
      </a>
    </li>
  </ul>
</DocumentFragment>
`;
